### PR TITLE
docs: Third party dependencies

### PR DESCRIPTION
**Pull request by tobi-or-not-tobi**
Added a section on how Oryx manage third party dependencies.

### CODE REVIEW

File: docs/scos/dev/front-end-development/202307.0/oryx/getting-started/oryx-packages.md [hunk 1]
```diff
@@ -82,3 +84,21 @@ The base layer contains packages that serve as utilities to all layers above. An
...
+
+- **Avoid shipping lock files in the boilerplate**: Lock files, such as `package-lock.json`, are not included in the Oryx boilerplate to allow application developers to benefit from the latest versions of dependencies when creating their projects.
+
+- **Utilize peer dependencies**: Oryx makes use of peer dependencies whenever possible, allowing application owners to choose specific versions of dependencies while maintaining compatibility with the framework.
+
+- **Documented version resolutions**: In cases where there are known security concerns or significant improvements, Oryx may document version resolutions for the `package.json` file for application owners to override specific dependency versions.
+
+By following these guidelines, Oryx aims to maintain a stable and secure ecosystem while providing flexibility for application developers to choose the versions of dependencies that best suit their needs.
```
Comment (tolerants) [inline docs/scos/dev/front-end-development/202307.0/oryx/getting-started/oryx-packages.md:104]: A few suggestion about possible points we can highlight: 
- we performing security audit (or should start perform it)
- we care about bundle size and performance
- we care about dependencies health checks

**Review by tolerants — CHANGES_REQUESTED** [review]

File: docs/scos/dev/front-end-development/202307.0/oryx/getting-started/oryx-packages.md [hunk 2]
```diff
@@ -1,29 +1,31 @@
...
 title: "Oryx: Packages"
 description: Use Oryx packages from npm to ensure you can easily upgrade to newer versions.
-last_updated: Apr 19, 2023
+last_updated: November 3, 2023
 template: concept-topic-template
 redirect_from:
   - /docs/scos/dev/front-end-development/202307.0/oryx/oryx-packages.html
 ---
 
+The Oryx code base is publicly [available on Github](https://github.com/spryker/oryx/), and published and distributed on [npmjs.com](https://www.npmjs.com/). Npm is a widely used registry of npm packages, that is used by package managers, like npm, yarn, deno, or bun, to install dependencies in an application.
```
[the remark this answers is not in the document]
Comment (tobi-or-not-tobi) [reply docs/scos/dev/front-end-development/202307.0/oryx/getting-started/oryx-packages.md:10]: Most popular depends on the users. There are other registries which are very popular as well. 
But happy to let Andrii have the last call on this section in general. Made a small  tweak.

File: docs/scos/dev/front-end-development/202307.0/oryx/getting-started/oryx-packages.md [hunk 3]
```diff
@@ -82,3 +84,21 @@ The base layer contains packages that serve as utilities to all layers above. An
...
+
+- **Use semantic versioning and caret (^) notation**: We prefer to use semantic versioning and the caret (^) notation in the `package.json` file to ensure compatibility and allow for seamless updates without introducing breaking changes. Ideally, versions only include the major version number, so that the minor and patch are controlled by application owners and can be updated over time. E.g. `"lit": "^2.0.0"`.
+
+- **Avoid shipping lock files in the boilerplate**: Lock files, such as `package-lock.json`, are not included in the Oryx boilerplate to allow application developers to benefit from the latest versions of dependencies when creating their projects.
+
+- **Utilize peer dependencies**: Oryx makes use of peer dependencies whenever possible, allowing application owners to choose specific versions of dependencies while maintaining compatibility with the framework.
+
+- **Documented version resolutions**: In cases where there are known security concerns or significant improvements, Oryx may document version resolutions for the `package.json` file for application owners to override specific dependency versions.
```
[the remark this answers is not in the document]
Comment (tobi-or-not-tobi) [reply docs/scos/dev/front-end-development/202307.0/oryx/getting-started/oryx-packages.md:102]: Reprased and made it consistent.

File: docs/scos/dev/front-end-development/202307.0/oryx/getting-started/oryx-packages.md (hunk 1 above)
Comment (tobi-or-not-tobi) [reply docs/scos/dev/front-end-development/202307.0/oryx/getting-started/oryx-packages.md:104]: rephrased

**Review by tolerants — APPROVED** [review]

**Pull request merged**